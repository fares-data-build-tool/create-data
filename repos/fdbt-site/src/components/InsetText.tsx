import React, { ReactElement } from 'react';

interface InsetTextProps {
    text: string;
}

const InsetText = ({ text }: InsetTextProps): ReactElement => {
    return <div className="govuk-inset-text">{text}</div>;
};

export default InsetText;
