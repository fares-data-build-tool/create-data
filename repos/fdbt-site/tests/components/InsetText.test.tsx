import { shallow } from 'enzyme';
import React from 'react';
import InsetText from '../../src/components/InsetText';

describe('InsetText', () => {
    it('should render the inset text', () => {
        const wrapper = shallow(<InsetText text="Snapshot text" />);
        expect(wrapper).toMatchSnapshot();
    });
});
