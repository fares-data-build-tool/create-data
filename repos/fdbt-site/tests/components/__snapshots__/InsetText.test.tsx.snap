// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InsetText should render the inset text 1`] = `
<div
  className="govuk-inset-text"
>
  Snapshot text
</div>
`;
